cfme/services/catalogs/catalog_items/__init__.py

```diff
@@ -161,6 +161,7 @@ class DetailsEntitiesCatalogItemView(View):
 class DetailsCatalogItemView(ServicesCatalogView):
     title = Text('#explorer_title_text')
     basic_info = SummaryForm("Basic Information")
+    currency_price = Text('.//div[../label[contains(text(), "Price / Month")]]')
```

Comment:
This xpath needs an adjustment, the `.//` is for a nested predicate, which I don't think you'll need here.  `../` will reference the parent node, but within the locator I don't think its necessary either.

`text()` shouldn't be used directly either, as its sensitive to leading/trailing whitespace. Check out the current uses:
https://github.com/ManageIQ/integration_tests/search?q=normalize-space&unscoped_q=normalize-space

`//div[label[contains(normalize-space(.), "Price / Month")]]` would work here, and will select the div containing the label and the div that contains the value.